declarations.d.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-empty-object-type */
+
+import type {
```

Comment:
Let's add some comments around this so people can understand what it does for us. 
Especially so we know it's a temporary hack until it's upgraded